reference/catalog/products_catalog.v3.yml

```diff
@@ -2950,6 +2952,9 @@ paths:
 
         **Read-Only Fields**
         * id
+
+        The Catalog API integrates with third-party Youtube, and videos must be loaded through YouTube at this time.
```

Comment:
```suggestion
        The Catalog API integrates with third-party YouTube, and you must load videos through YouTube now.
```

Reply:
same here, I reworded your suggestion to: 

`The Catalog API integrates with third-party YouTube, and you must load videos through YouTube.`